novawallet/Common/Model/KeystoreTag/KeystoreTag+Gift.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+extension KeystoreTagV2 {
+    static func substrateSecretKeyTagForGift(
+        accountId: AccountId? = nil
```

Comment:
Do we really have cases where `accountId` is nil? If not then no need to make them optional